app/views/pageflow/favicons/_entry.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if icon_path.present? %><link rel='icon' href='<%= icon_path %>' type='image/ico'><% end %>
+<% if icon_32_path.present? %><link rel='icon' type='image/png' sizes='32x32' href='<%= icon_32_path %>'><% end %>
+<% if icon_16_path.present? %><link rel='icon' type='image/png' sizes='16x16' href='<%= icon_16_path %>'><% end %>
+
+<% if apple_touch_icon_path.present? %><link rel='apple-touch-icon' sizes='180x180' href='<%= apple_touch_icon_path %>'><% end %>
+<% if webmanifest_path.present? %><link rel='manifest' href='<%= webmanifest_path %>'><% end %>
+<% if safari_pinned_tab_path.present? %><link rel='mask-icon' href='<%= safari_pinned_tab_path %>' color='#5bbad5'><% end %>
```

Comment:
**question**: In which case will these conditions be `false`?

Reply:
At first thought, I thought if there is no file then it should return false and the tag won't be rendered. But, if there isn't a file then it returns an error. Removed these.